README.md

```diff
@@ -76,13 +76,13 @@ where app.json may look something like:
 }
 ```
 
-You can specify a specific node_modules top level path by using the `-p` flag.
+You can specify a specific path to be passed to Glue as the `relativeTo` option by using the `-p` flag.
 
 ```javascript
-rejoice -c app.json -p /full/path/node_modules
+rejoice -c app.json -p /full/path/to/project/plugin/dir
```

Comment:
Correct, when you don't specify the `-r` flag this path is passed as is to Glue `{ relativeTo: '...' }`.  But in that case I don't believe you would use a node_modules folder.  Or perhaps it's just my specific use case where my modules aren't located under `node_modules/myplugin`, but instead `lib/node_modules`.  If the broader case is they would be located in a node_modules directory then I would agree it needs updating.